app-api/routes/trace.py

```diff
@@ -294,6 +295,16 @@ async def analyze_trace(
         debug_options=analysis_request.options.debug_options,
     )
 
+    async def sse_parse(chunks):
+        """Function to parse sse chunks."""
+        output = b""
+        async for chunk in chunks:
+            output += chunk
+            if b"\n\n" in output:
+                lines = output.split(b"\n\n")
+                for line in lines[:-1]:
+                    yield line.decode("utf-8")
+                output = lines[-1]
```

Comment:
Could this miss the very last chunk, e.g. if there is no \n\n in the very end of a stream?

Reply:
Maybe, we can add one last parsing attempt after the loop here.